torch/quantization/quantize.py

```diff
@@ -155,7 +155,7 @@ def insert_activation_post_process(m, special_act_post_process=None):
             setattr(module, name, observed_child)
             # TODO: These are the modules that cannot be observed
             #       Once there are more, we should move them to a separate list
-            if custom_module_class_mapping[type(child)] != nnqa.LSTM:
+            if custom_module_class_mapping[type(child)] not in [nnqa.LSTM, nnqa.MultiheadAttention]:
```

Comment:
should we move this into a list in the config?

Reply:
sounds great, would it be ok to do this in this PR?

Reply:
After thinking a moment -- I will send another PR with the mappings stacked on top of this. I think it's a bad idea to piggyback new features with the existing stuff because in case of a bisect, the whole thing might go down.

Reply:
sg

Reply:
> I will send another PR with the mappings stacked on top of this

I'm ok with not doing it here, but for the future would be great to consider stacking the one-off fixes before the large logic change PR (instead of after), to reduce the amount of time the hacks live in the master branch to zero.